Minus Icon (#1045)
sumup-oss/circuit-ui: .changeset/heavy-toys-thank.md

---

> '@sumup/icons': major

Reviewer: ```suggestion
'@sumup/icons': minor
```

---

> Added a new asset to the icons package. The 'minus_small' icon.

Reviewer: This will be added to the changelog for `@sumup/icons`, so the context is implied.

```suggestion
Added a new minus icon in a small size.
```
